### PR TITLE
Russian Elbrus processors support

### PR DESCRIPTION
**Pull request by beldmit**
##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: config [hunk 1]
```diff
@@ -551,6 +551,7 @@ case "$GUESSOS" in
   pentium-*-vxworks*) OUT="vxworks-pentium" ;;
   simlinux-*-vxworks*) OUT="vxworks-simlinux" ;;
   mips-*-vxworks*) OUT="vxworks-mips";;
+  e2k*) OUT="linux-generic64 -DL_ENDIAN" ;;
```
Comment (levitte) [inline config]: Will only Linux be implemented for those processors, ever?  If you can predict the future that well, this is fine, but if not, the case should be `e2k-*-linux*`

Comment (beldmit) [reply config]: I hardly can imagine that any not Linux-based OS will appear in Russia more or less soon.

Comment (beldmit) [reply config]: Well, I was wrong. @levitte, many thanks!

**Comment by beldmit** [comment]
@paulidale, could you please reapprove?

**Comment by beldmit** [comment]
Applied. Thanks!

**Pull request closed without merging**